src/main/java/uk/gov/justice/hmpps/prison/service/AgencyService.java

```diff
@@ -181,6 +181,7 @@ public Page<Agency> getAgencies(final long offset, final long limit) {
     }
 
     public List<Agency> findAgenciesByUsername(final String username) {
+        if (StringUtils.isBlank(username)) return Collections.emptyList();
```

Comment:
this was coming through as `null` under some circumstances causing
```
Invalid key type, expected : java.lang.String but was : org.springframework.cache.interceptor.SimpleKey
```
exception in the logs